app.py

```diff
@@ -123,19 +123,15 @@ def pop_host_from_current_request(endpoint, values):
     # Security configuration
     csrf.init_app(flask_app)
 
-    # TODO: Work out which scripts these SHAs relate to and comment this better
-    csp = {
-        "default-src": "'self'",
-        "script-src": [
-            "'self'",
-            "'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='",
-            "'sha256-l1eTVSK8DTnK8+yloud7wZUqFrI0atVo6VlC6PJvYaQ='",
-            "'sha256-GUQ5ad8JK5KmEWmROf3LZd9ge94daqNvd8xy9YS1iDw='",
-            "'sha256-ndKdvEBfUn27+cpVrq2H697Dg88x3tsepe3veTUtsaA='",
-            "'sha256-oR+vHsLl1DMudaF9Ay6TIDK2lXwFka0z5sbdN5RZQxE='",
```

Comment:
Any idea where these scripts are? Are they in templates we import? Did we just forget to delete them?

Only script I see we are directly importing is `govuk-frontend/govuk-frontend-5.4.0.min.js` which should be covered by `self`.

Reply:
The most common example we have is those being the sha256's for these blocks:

```
  <script type="module">
      import { initAll } from '{{ govukFrontendJsURI }}'
      initAll()
  </script>
```

Which are now being nonced